### PR TITLE
Update JSON schema with Fixed Values in grammar

### PR DESCRIPTION
**Pull request by Rebecca-Qian**
# Description

As in title.

Fixes # (issue)

#191 

Please check the options that are relevant.

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [ ] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change

## Type of requested review

- [ ] I want a thorough review of the implementation.
- [x] I want a high level review. 
- [ ] I want a deep design review.

# Testing

```
python ~/droidlet/craftassist/test/validate_json.py
```

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.

### CODE REVIEW

File: tools/data_processing/preprocess_datasets_for_autocomplete.py [hunk 1]
```diff
@@ -26,7 +26,6 @@ def create_JSON_from_txt(annotations_dir_path):
 
         for line in data:
             if "|" in line:
-                command, action_dict = line.strip().split("|")
```
Comment (kavyasrinet) [inline tools/data_processing/preprocess_datasets_for_autocomplete.py:29]: are we only writing dictionaries now ?

File: base_agent/documents/json_schema/filters.schema.json [hunk 2]
```diff
@@ -15,9 +15,7 @@
                 "ordinal": {
                     "anyOf": [
                         {
-                            "enum": [
-                                "FIRST"
-                            ]
+                            "$ref": "action_dict_components.schema.json#/definitions/fixed_value"
```
Comment (kavyasrinet) [inline base_agent/documents/json_schema/filters.schema.json:18]: assuming changes here are in sync with : https://github.com/facebookresearch/droidlet/issues/191

**Review by kavyasrinet — APPROVED** [review]
LGTM if you have already validated against datasets.

File: base_agent/documents/json_schema/filters.schema.json (hunk 2 above)
Comment (Rebecca-Qian) [reply base_agent/documents/json_schema/filters.schema.json:18]: Yep!

**Pull request merged**